spec/core_functions/selector/is_superselector/simple/pseudo/no_arg.hrx

```diff
@@ -0,0 +1,57 @@
+<===> class/equal/input.scss
+a {b: is-superselector(":c", ":c")}
+
+<===> class/equal/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> class/unequal/input.scss
+a {b: is-superselector(":c", ":d")}
+
+<===> class/unequal/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> class/and_element/input.scss
+a {b: is-superselector(":c", "::c")}
+
+<===> class/and_element/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> element/equal/input.scss
+a {b: is-superselector(":c", ":c")}
+
```

Comment:
Duplicated above in class/equal/input.scss

Reply:
Done.